core/primitives/src/shard_layout.rs

```diff
@@ -61,10 +61,12 @@ impl ShardLayout {
         Self::v0(1, 0)
     }
 
+    /// Return a V0 Shardlayout
     pub fn v0(num_shards: NumShards, version: ShardVersion) -> Self {
         Self::V0(ShardLayoutV0 { num_shards, version })
     }
 
+    /// Return a V1 Shardlayout
```

Comment:
we might also want to mention (on top of this file) where is the current ShardLayout configured (the one that we use in mainnet and/or testnet)